neurobooth_os/iout/stream_utils.py

```diff
@@ -10,6 +11,13 @@ class DataVersion(NamedTuple):
     def __str__(self):
         return f'{self.major}.{self.minor}'
 
+    @staticmethod
+    def from_str(version_str: str) -> 'DataVersion':

```

Comment:
This method will be used by the next phase. It was left over from when I trimmed out subsequent work to make smaller PRs.